src/plugins/data_views/docs/openapi/entrypoint.yaml

```diff
@@ -15,35 +15,35 @@ servers:
   - url: /
 paths:
 # Default space
-  '/api/data_views':
-    $ref: 'paths/api@data_views.yaml'
-  '/api/data_views/data_view':
-    $ref: 'paths/api@data_views@data_view.yaml'
-  '/api/data_views/data_view/{viewId}':
-    $ref: 'paths/api@data_views@data_view@{viewid}.yaml'
-  '/api/data_views/data_view/{viewId}/fields':
-    $ref: 'paths/api@data_views@data_view@{viewid}@fields.yaml'
-  '/api/data_views/data_view/{viewId}/runtime_field':
-   $ref: 'paths/api@data_views@data_view@{viewid}@runtime_field.yaml'
-  '/api/data_views/data_view/{viewId}/runtime_field/{fieldName}':
-    $ref: 'paths/api@data_views@data_view@{viewid}@runtime_field@{fieldname}.yaml'
-  '/api/data_views/default':
-    $ref: 'paths/api@data_views@default.yaml'
+  # '/api/data_views':
```

Comment:
Does this maintain the default space apis? It looks like they've been disabled.

Reply:
I haven't maintained the default space APIs, they're not defined in any of the other Kibana API specs I've used and mean duplicating all the operations. Happy to do the work to have both defined, but it's not clear what the benefit is.

Reply:
@tobio Seems like thats probably worth a wider conversation, something kibana core in particular should be on board with. 

It does change the way I think about the API but I don't really have a strong opinion.